hack-sound-server/meson.build

```diff
@@ -0,0 +1,48 @@
+namespace = 'HackSoundServer'
+library_name = 'hacksoundserver'
+export_package_name = 'hack-sound-server'
+
+api_version = '0'
+api_name = '@0@-@0@'.format(library_name, api_version)
+libtool_version = '0.0.0'
+
+hacksoundserver_dbus_sources = gnome.gdbus_codegen(
+    export_package_name,
+    sources: 'data/com.endlessm.HackSoundServer.xml',
+    interface_prefix : 'com.endlessm.',
+    autocleanup: 'all',
```

Comment:
This feature requires Meson 0.47.0, so you should put that requirement in the initial `project()` call.